### PR TITLE
chore: delete references to 'registry'

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Happened to have a look at `DEVELOPMENT` and noticed we left some baggage behind 🎒

### CODE REVIEW

**Review by xla — APPROVED** [review]
⛺️ 🌛 🙌 📹

**Review by MeBrei — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -124,10 +124,10 @@
...
     "proxy:build:release": "cd proxy && cargo build --release --all-features --all-targets",
     "proxy:clean": "cd proxy && cargo clean",
-    "proxy:start": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=emulator",
-    "proxy:start:test": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=emulator --test",
-    "proxy:start:ffnet": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=rpc.ff.radicle.network",
-    "proxy:start:devnet": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --registry=35.241.138.91",
+    "proxy:start": "cd proxy && cargo build --bin git-remote-rad && cargo run",
+    "proxy:start:test": "cd proxy && cargo build --bin git-remote-rad && cargo run -- --test",
+    "proxy:start:ffnet": "cd proxy && cargo build --bin git-remote-rad && cargo run",
+    "proxy:start:devnet": "cd proxy && cargo build --bin git-remote-rad && cargo run",
```
Comment (rudolfs) [inline package.json:130]: Shouldn't we remove these altogether?

Comment (NunoAlexandre) [reply package.json:130]: indeed :mag:

Comment (NunoAlexandre) [reply package.json:130]: And a bunch more actually. Thanks for your attentive look :ok_hand:

Comment (xla) [reply package.json:130]: I think he meant to delete the `ffnet` and `devnet` comamnds alltogether.

**Review by xla — APPROVED** [review]
😜 🌕 🉐 🎵

**Review by rudolfs — APPROVED** [review]

**Pull request merged**